### PR TITLE
Get all the Published Quizzes

### PR DESCRIPTION
**Pull request by aiman-syeda**
- Added functionality "Get all the published Quizzes except quizzes, created by the user itself".
- Also added request and saved example to Postman collection.
- Verified all the other functionalities and updated change password saved example on Postman collection.
- Added category field to the constant quiz in createQuiz controller function.

### CODE REVIEW

File: backend/src/routes/quiz.ts [hunk 1]
```diff
@@ -63,6 +64,9 @@ router.post(
   createQuiz
 );
 
+//Get  quiz/allpublished quiz
+router.get("/allPublishedQuiz",isAuthenticated, getallQuiz);
```
Comment (vermaaakarsh) [inline backend/src/routes/quiz.ts]: there should be no caps in url

File: backend/src/controllers/quiz.ts [hunk 2]
```diff
@@ -12,9 +12,9 @@ const createQuiz: RequestHandler = async (req, res, next) => {
     const category = req.body.category;
     const questionList = req.body.questionList;
     const answers = req.body.answers;
-    const passing_percentage=req.body.passing_percentage;
-    
-    const quiz = new Quiz({ name, questionList, answers, passing_percentage, createdBy });
+    const passing_percentage = req.body.passing_percentage;
```
Comment (vermaaakarsh) [inline backend/src/controllers/quiz.ts]: change everything to camelCase.

**Review by vermaaakarsh — CHANGES_REQUESTED** [review]

File: backend/src/controllers/quiz.ts (hunk 2 above)
Comment (aiman-syeda) [reply backend/src/controllers/quiz.ts]: Is it for 2 word variables?

File: backend/src/routes/quiz.ts (hunk 1 above)
Comment (aiman-syeda) [reply backend/src/routes/quiz.ts]: I'll make the corrections

File: backend/src/controllers/quiz.ts (hunk 2 above)
Comment (vermaaakarsh) [reply backend/src/controllers/quiz.ts]: passing_percentage should be passingPercentage and likewise

Comment (aiman-syeda) [reply backend/src/controllers/quiz.ts]: Got it Sir

**Comment by aiman-syeda** [comment]
@vermaaakarsh Sir, I have implemented and verified the changes accordingly.

**Pull request merged**